tests/appsec/rasp/test_lfi.py

```diff
@@ -2,7 +2,7 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2021 Datadog, Inc.
 
-from utils import features, weblog, interfaces, scenarios, rfc
+from utils import features, weblog, interfaces, scenarios, rfc, bug
```

Comment:
minor: don't see why bug is needed here?

Reply:
Not needed. I will delete it now. Thanks